css-canvas-medical/styles.css

```diff
@@ -0,0 +1,246 @@
+.container {
+  margin: auto;
+  width: 1000px;
+}
+
+h1,
+h2,
+h3 {
+  font-family: "Tiempos Headline", sans-serif;
+  color: #6e6e6e;
+}
+
+p,
+a {
+  font-family: Avenir, sans-serif;
+  font-size: 10px;
+  line-height: 1.5;
+}
+
+button {
+  background-color: #294364;
+  color: white;
+  border-radius: 3px;
+  border: none;
+  padding: 10px 30px;
+}
+
+.row {
+  display: flex;
+}
+
+.column-100 {
+  flex-basis: 100%;
+}
+
+.test1 {
+  border-bottom: 2px solid lightgrey;
+}
+
+.links {
+  position: relative;
+  left: 60%;
+  color: #808080;
+  text-decoration: none;
+}
+
+hr {
+  border: 0.1px solid lightgray;
+  margin-top: 15px;
+}
+
+.test {
+  color: #808080;
+  text-decoration: none;
+  padding-right: 10px;
+}
+
+.rowi {
+  border-bottom: 1px solid rgb(220 219 219);
+}
+
+.column-image {
+  flex-basis: 100%;
+}
+
+.overlay {
+  position: absolute;
+  top: 30px;
+  right: 50px;
+  width: 90%;
+  height: 93.6%;
+  background: rgb(90 90 90 / 50%);
+}
+
+.blue-logo {
+  position: relative;
+  width: 100px;
+  top: 30px;
+  left: 50px;
+}
+
+/* .column-2 {
+  display: flex;
+  justify-content: end;
+  align-items: center;
+  padding: 0 50px 10px;
+} */
+
+.column-2 {
+  flex-basis: auto;
+}
+
+.column-2 a {
+  color: #808080;
+  text-decoration: none;
+  padding: 10px;
+}
+
+.textok {
+  position: absolute;
+  top: 20px;
+  font-weight: normal;
+}
+
+.row4 {
+  display: flex;
+  text-align: center;
+}
+
+.testone {
```

Comment:
Numbered classes are not good as names, they don't describe their purpose at all.